wiki/Tournaments/ASC/2/en.md

```diff
@@ -0,0 +1,208 @@
+| **nagisa fanboys** | **5** | 0 | Harikitte Ikou! | [#1](https://osu.ppy.sh/community/matches/119842053) |
+| **BALLS** | **5** | 2 | is that team name | [#1](https://osu.ppy.sh/community/matches/119842382) |
+| **Skill Issue** | **5** | 2 | TEAM SEVEN | [#1](https://osu.ppy.sh/community/matches/119842569) |
+| Foreclosure Task Rob | 4 | **5** | **Capoo Fans** | [#1](https://osu.ppy.sh/community/matches/119843477) |
+| wedoknowwhatbye | 2 | **5** | **The Maple Simpers** | [#1](https://osu.ppy.sh/community/matches/119843940) |
+
+### Qualifiers
+
+Detailed statistics for this round can be found in [this web page](https://asia-suiji-cup.vercel.app/tournament/2/stats) alongside the final standings on which teams sets the highest average rank and average score.
+
+## Ruleset
+
+### General rules
+
+1. This is a **3v3 double-elimination tournament**, with a team size of 6 and a total of 16 teams. Only the top 96 players will be chosen to join this edition.
+2. The tournament is open to all players who have **country flags from Asia and Oceania** in their profiles. Participants must have flags from these regions to be eligible for entry.
+3. To sign up players must do so through the website and must be in the discord server. Failure to do so will count your registration as incomplete and not eligible to participate.
+4. Players display names in the Discord server must be the same as their osu! username during the entire duration of the tournament.
+5. A "sandbagging" formula will be used to help determine the true ranks of players to avoid disruptions and highly unbalanced teams that may potentially ruin the Suiji Cup experience. Player seedings and formula details will be revealed right before sign ups close.
+6. Participants will be split up into 3 different seeds, "A", "B" and "C". Each team will be formed by a live randomization process and will consist of 2 players from each seed (a team will have 6 players, A, A, B, B, C, C).
+7. "A" seeded players have the ability to substitute with "B" or "C" seed on their team. "B" seeded players also have the ability to substitute but only with "C" seed on their team.
+8. Teams will be given 5 days to choose a team name and a captain. Failure to submit a team name and a captain will result in an assigned team name chosen by the staff and the highest-ranked player being made captain by default.
+
+### Additional information
+
+1. Unlike the previous iteration, there will only be 1 SV pick in the mappool, and it cannot be banned.
+2. Visual mods such as: HD, FL, FI, and MR along with NF mod are allowed. Meanwhile, EZ and HR mod are not allowed.
+3. In the event of a disconnection, the map can be replayed if the disconnect is within the first 30 seconds or before the first quarter of the map. Otherwise, if there is proof of the disconnected player's score, it will be used. Each team is allowed one replay per match, the referee will have the final say in disconnection situations.
```

Comment:
```suggestion
3. In the event of a disconnection, the map can be replayed if the disconnect is within the first 30 seconds or before the first quarter of the map. Otherwise, if there is proof of the disconnected player's score, it will be used. Each team is allowed one replay per match, and the referee will have the final say in these situations.
```